cypress/integration/sitemap-vrt/constants.ts

```diff
@@ -26,6 +26,7 @@ export const SITEMAP = [
   '/components/chat-log/',
   '/components/checkbox/',
   '/components/data-grid/',
+  '/components/detail-text',
```

Comment:
nit(non-blocking): is it ok that the url doesn't have the trailing slash at the end?

Reply:
Yeah it ok, it showed up in the vrt.